From edf0f10bc079e5e3922bddbb8185dc626ab14a1b Mon Sep 17 00:00:00 2001
From: Emil Velikov <emil.l.velikov@gmail.com>
Date: Sun, 10 Oct 2021 12:28:16 +0100
Subject: [PATCH] dkms.conf: remove REMAKE_INITRD option

The default state of the option is no - so let's just remove it.
This aligns with my plan of deprecating the option in upstream dkms.

Signed-off-by: Emil Velikov <emil.l.velikov@gmail.com>
---
 dkms.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dkms.conf b/dkms.conf
index 4e916fc..b1a0069 100644
--- a/dkms.conf
+++ b/dkms.conf
@@ -8,5 +8,4 @@ CLEAN="make clean"
 BUILT_MODULE_NAME[0]="$PACKAGE_NAME"
 DEST_MODULE_LOCATION[0]="/extra"
 
-REMAKE_INITRD="no"
 AUTOINSTALL="yes"
